app/components/billing-status-display/billing-status-badge.hbs

```diff
@@ -0,0 +1,25 @@
+<div ...attributes>
+  {{#if this.shouldShowVipBadge}}
+    <Header::VipBadge @size={{@size}} />
+  {{else if this.shouldShowMemberBadge}}
+    <Header::MemberBadge @size={{@size}} />
+  {{else if this.activeDiscountForYearlyPlan}}
```

Comment:
```suggestion
  {{else if (and this.activeDiscountForYearlyPlan this.canSeeDiscountCoundown)}}
```

(This way, if the user can't see the discount countdown, the next condition will trigger - else we won't end up rendering the upgrade button for many users)